codility_training/lessons.lesson03.TimeComplexity.FrogJmp/sangmandu.py

```diff
@@ -0,0 +1,15 @@
+# you can write to stdout for debugging purposes, e.g.
+# print("this is a debug message")
+
+def solution(X, Y, D):
+    # write your code in Python 3.6
+    pass
+    return (0 if (Y-X)%D == 0 else 1) + ((Y-X)//D if X < Y else 0)
+
+'''
+ X = 10
+ Y = 85
+ D = 30
+
+ return 3
+ '''
```

Comment:
```suggestion
print(
    solution(10, 85, 30) == 3,
    # solution(X, Y, D) == result,
)
```
이런식으로 코드상에서 테스트해볼 수 있도록 작성하는걸 권장합니다~